python/vyos/util.py

```diff
@@ -46,6 +49,20 @@ def debug_msg(message, flag=''):
     if debug(flag):
         print(f'DEBUG/{flag:<6} {message}')
 
+    if not debug('developer'):
+        return
+
+    logfile = '/tmp/full-log'
+    existed = os.path.exists(logfile)
+
+    with open(logfile, 'a') as f:
+        f.write(f'DEBUG/{flag:<6} {message}\n')
+        if not existed:
+            # at boot the file is created as root:vyattacfg
+            # at runtime the file is created as user:vyattacfg
+            # do not use run/cmd to not have a recursive call to this code
+            os.system(f'sudo chmod g+w {logfile}')
```

Comment:
Why you use always sudo? It's not required

Reply:
ok will remove.